README.md

```diff
@@ -0,0 +1,3 @@
+# cairo-fb-examples
+
```

Comment:
Am Freitag, den 08.01.2016, 19:31 -0800 schrieb Stefan Agner:
> > @@ -0,0 +1,3 @@
> > +# cairo-fb-examples
> > +
> 
> Naaah, please..

Why not?
Max

> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/toradex/cairo-fb-examples/pull/1/files#r49259670